### PR TITLE
Feat: Migrate to helios 0.2.0

### PR DESCRIPTION
**Pull request by danilowhk**
Update Beerus to use Beerus 0.2.0 which is wasmable.

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

Issue Number: N/A

# What is the new behavior?

-
-
-

# Does this introduce a breaking change?

- [ ] Yes
- [ ] No

# Other information

### CODE REVIEW

File: beerus_core/src/config.rs [hunk 1]
```diff
@@ -42,13 +45,15 @@ impl Config {
         let starknet_core_contract_address = std::env::var("STARKNET_CORE_CONTRACT_ADDRESS")
             .unwrap_or_else(|_| DEFAULT_STARKNET_CORE_CONTRACT_ADDRESS.to_string());
         let starknet_core_contract_address = Address::from_str(&starknet_core_contract_address)?;
+        let data_dir = PathBuf::from("/tmp");
```
Comment (drspacemn) [inline beerus_core/src/config.rs:48]: Is the desired result that this is set by the user? If so we need to check an environment variable is set here.

Or can default to "/tmp" if none is set.

File: beerus_core/tests/config.rs [hunk 2]
```diff
@@ -245,12 +247,13 @@ mod tests {
     #[test]
     fn given_ethereum_network_is_invalid_when_ethereum_network_then_returns_error() {
         let config = Config {
-            ethereum_network: "invalid".to_string(),
+            ethereum_network: "mainnet2".to_string(),
```
Comment (drspacemn) [inline beerus_core/tests/config.rs:250]: "invalid".to_string() is more explicit here as "mainnet2" isn't obviously a wrong network name upon reading.

Comment (danilowhk) [reply beerus_core/tests/config.rs:250]: Ok!

File: beerus_core/src/config.rs (hunk 1 above)
Comment (danilowhk) [reply beerus_core/src/config.rs:48]: Ok! Will move to .env

**Review by drspacemn — APPROVED** [review]

**Pull request merged**